DEVELOPERS.md

```diff
@@ -63,7 +63,7 @@ Before you start a development server, you must choose if you want to work on th
 1. Go to the [Supabase Website](https://supabase.com) directory
 
    ```sh
-   cd apps/www
+   cd apps
```

Comment:
This change does not seem to be correct, according to the section title if you wanna work on the website, you have to work from the apps/www folder.